_posts/blog/2017-07-18-dwh-part2-we-need-a-storage.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: "A Journey Towards a Custom Data Warehouse Solution<br/> Part 2: We Need a Storage"
+description: 
+modified:
+categories: blog
+author: robert_bordo
+excerpt: We would like to share our second part towards a custom data warehouse solution. This time it's all about selecting a proper storage.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake, Storage, Redshift, S3, Snowflake]
+image:
+date: 2017-07-18T00:39:55-04:00
+---
+
+> In the beginning we created a cluster. And the cluster was without form, and void; and nulls were upon the face of the storage.
+
+As we learned in [part 1 of our series](../dwh-part1_getting_started), a data warehouse consists of several components. 
+The key component is the storage. All the others group around it. But how can one draw a decision which storage solution to take? What is out there anyway? 
+
+## Preface
+
+In a perfect world there would be only one kind of storage that fits all the needs of current DWH development and analysis. 
+But since we are not living in that kind of place, we have several options. And its number increases, the deeper one dives into the topic. 
```

Comment:
"But since we are not living in that kind of place, we have several options. And the  number of options increase the deeper one dives into the topic."